PKGBUILD

```diff
@@ -203,7 +211,7 @@ _package-headers() {
   install -Dt "$builddir/rust" -m644 scripts/target.json
   install -Dt "$builddir/rust" -m644 rust/*.rmeta
   install -Dt "$builddir/rust" -m644 rust/*.so
-  install -Dt "$builddir" -m644 ../rust-toolchain
+  install -Dt "$builddir" -m644 "$src_dir/rust-toolchain"
 
```

Comment:
This should be `$srcdir`, then it works perfectly for me.

Reply:
Thanks I noticed this myself in the meantime, since the packaging step failed in my test run. It's fixed now.